modules/selecting-cluster-capabilities.adoc

```diff
@@ -0,0 +1,22 @@
+// Module included in the following assemblies:
+//
+// *installing/cluster-capabilities.adoc
+
+:_content-type: PROCEDURE
+[id="selecting-cluster-capabilities_{context}"]
+= Selecting cluster capabilities
+You can select cluster capabilities by following one of the installation methods that allow you to customize your cluster, such as "Installing a cluster on AWS with customizations" or "Installing a cluster on GCP with customizations". 
```

Comment:
Pondering IBMSG's guidance about avoiding "allow". Maybe:

s/that allow you to customize your cluster/that include cluster customizations/